numpy/linalg/linalg.py

```diff
@@ -99,7 +97,7 @@ def _raise_linalgerror_svd_nonconvergence(err, flag):
     raise LinAlgError("SVD did not converge")
 
 def get_linalg_error_extobj(callback):
-    extobj = list(_linalg_error_extobj)
+    extobj = list(_linalg_error_extobj) # make a copy
```

Comment:
Nit: PEP8 says two spaces before the `#`

Reply:
@eric-wieser  Addressed it.